specs/decimal/decimal.md

```diff
@@ -68,125 +229,106 @@ namespace Microsoft.Windows.Foundation
-
-        void SetFromBoolean(Boolean value);
-        void SetFromInt16(Int16 value);
-        void SetFromInt32(Int32 value);
-        void SetFromInt64(Int64 value);
-        void SetFromUInt8(UInt8 value);
-        void SetFromUInt16(UInt16 value);
-        void SetFromUInt32(UInt32 value);
-        void SetFromUInt64(UInt64 value);
-        void SetFromSingle(Single value);
-        void SetFromDouble(Double value);
-        void SetFromString(String value);                           // LCID=LOCALE_INVARIANT
-        void SetFromStringWithSystemDefaultLocale(String value);    // LCID=GetSystemDefaultLCID()
-        void SetFromStringWithUserDefaultLocale(String value);      // LCID=GetUserDefaultLCID()
-        void SetFromStringWithThreadLocale(String value);           // LCID=GetThreadLocale()
-        void SetFromStringWithInvariantLocale(String value);        // LCID=LOCALE_INVARIANT
-        void Set(IInspectable value);
-        void SetFromDecimal(Decimal value);
-        void SetFromDecimalValue(DecimalValue value);
-
-        Boolean ToBoolean();
-        Int16 ToInt16();
-        Int32 ToInt32();
-        Int64 ToInt64();
-        UInt8 ToUInt8();
-        UInt16 ToUInt16();
-        UInt32 ToUInt32();
-        UInt64 ToUInt64();
-        Single ToSingle();
-        Double ToDouble();
-        //String ToString(); inherited from IStringable     // LCID=LOCALE_INVARIANT
-        String ToStringWithSystemDefaultLocale();           // LCID=GetSystemDefaultLCID()
-        String ToStringWithUserDefaultLocale();             // LCID=GetUserDefaultLCID()
-        String ToStringWithThreadLocale();                  // LCID=GetThreadLocale()
-        String ToStringWithInvariantLocale();               // LCID=LOCALE_INVARIANT
-        IInspectable ToObject();
-        Decimal ToDecimal();                //TODO: Rename to Copy(value) or Clone(value) ?
-        DecimalValue ToDecimalValue();
-
-        /// Return true if (this == value).
-        Boolean Equals(Decimal value);
-
-        /// Compare this decimal with value.
-        /// @return 0 if this and value are equal, <0 if this is less than value or >0 if this is greater than value.
-        Int32 Compare(Decimal value);
+        static DecimalValue FromBoolean(Boolean value);
+        static DecimalValue FromInt16(Int16 value);
+        static DecimalValue FromInt32(Int32 value);
+        static DecimalValue FromInt64(Int64 value);
+        static DecimalValue FromUInt8(UInt8 value);
+        static DecimalValue FromUInt16(UInt16 value);
+        static DecimalValue FromUInt32(UInt32 value);
+        static DecimalValue FromUInt64(UInt64 value);
+        static DecimalValue FromSingle(Single value);
+        static DecimalValue FromDouble(Double value);
+        static DecimalValue FromString(String value);                          // LCID=LOCALE_INVARIANT
+        static DecimalValue FromStringWithSystemDefaultLocale(String value);   // LCID=GetSystemDefaultLCID()
```

Comment:
Is there a version that the app can pass in a parameter?

Reply:
RECOMMEND: Per other comment DON'T expose LCID. Replace ctor and oper= with the trio of named methods

* ToString()
* ToStringInvariant()
* ToStringWithLocale(string localeName)

(and the inverse From... equivalents)